Sources/eppo/EppoClient.swift

```diff
@@ -52,52 +52,68 @@ public class EppoClient {
         let (urlData, _) = try await httpClient.get(url);
         self.flagConfigs = try JSONDecoder().decode(FlagConfigJSON.self, from: urlData);
     }
-    
-    public func getAssignment(
-        _ subjectKey: String,
-        _ flagKey: String,
-        _ subjectAttributes: SubjectAttributes) throws -> String?
-    {
-        return try getInternalAssignment(subjectKey, flagKey, subjectAttributes, false)?.stringValue()
-    }
 
     public func getBoolAssignment(
-        _ subjectKey: String,
-        _ flagKey: String,
-        _ subjectAttributes: SubjectAttributes = SubjectAttributes()) throws -> Bool?
+        flagKey: String,
+        subjectKey: String,
+        subjectAttributes: SubjectAttributes = SubjectAttributes(),
```

Comment:
We talked about subject attributes being required - still if anything is going to have a default argument its this. Should we put it to bed and just avoid the default (requiring users to specify the empty object?)

Reply:
I think both paths are reasonable -- but maybe for consistency across all SDKs we force it to be specified always. 

In particular, if we do not hten it may be confusing for JSON objects, as subjectAttributes and the default are both dictionaries..., so if only 1 is specified it's really not obvious which argument you are supplying

Reply:
It's only confusing for dynamically typed languages 😌